data/src/main/java/com/example/data/db/tables/LastTenChosenCitiesEntity.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.data.db.tables
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity
+class LastTenChosenCitiesEntity(
+    @PrimaryKey(autoGenerate = false)
+    val currentTime: Long,
```

Comment:
The name of the field is confusing. Actually, the field doesn't contain current time. It contains lastUsedTime or lastChosenTime or something like this. 
And we already have key in this type of data. It is cityID. Let's use this field as a key.

Reply:
Fixed.
I cannot use cityID as a key because the data in the table is sorted in ascending order by cityID. It breaks my algorithm for removing the extra 11-th city. I need every new addition of a city to be in the last line of the table